rotkehlchen/db/schema.py

```diff
@@ -782,3 +782,57 @@
 COMMIT;
 PRAGMA foreign_keys=on;
 """
+
+# Minimized version of the db schema that is used for db schema sanity checks.
+# Should be autogenerated by tools/scripts/generate_simplified_db_schema.py
+MINIMIZED_ENCRYPTED_DB_SCHEMA = {
+    'trade_type': 'typeCHAR(1)PRIMARYKEYNOTNULL,seqINTEGERUNIQUE',
+    'location': 'locationCHAR(1)PRIMARYKEYNOTNULL,seqINTEGERUNIQUE',
+    'asset_movement_category': 'categoryCHAR(1)PRIMARYKEYNOTNULL,seqINTEGERUNIQUE',
+    'balance_category': 'categoryCHAR(1)PRIMARYKEYNOTNULL,seqINTEGERUNIQUE',
+    'assets': 'identifierTEXTNOTNULLPRIMARYKEY',
+    'timed_balances': 'categoryCHAR(1)NOTNULLDEFAULT("A")REFERENCESbalance_category(category),timestampINTEGER,currencyTEXT,amountTEXT,usd_valueTEXT,FOREIGNKEY(currency)REFERENCESassets(identifier)ONUPDATECASCADE,PRIMARYKEY(timestamp,currency,category)',  # noqa: E501
+    'timed_location_data': 'timestampINTEGER,locationCHAR(1)NOTNULLDEFAULT("A")REFERENCESlocation(location),usd_valueTEXT,PRIMARYKEY(timestamp,location)',  # noqa: E501
```

Comment:
I would 100% put this in a different file. This file is huge already.
What's more I would ignore the noqa: E501 rule there in the entire file